src/states/ChatState.cpp

```diff
@@ -65,6 +68,34 @@ void ChatState::setLastActivityText(const QString &text)
     emit lastActivityTextChanged(text);
 }
 
+bool ChatState::isAdmin() const
+{
+    return m_isAdmin;
+}
+
+void ChatState::setIsAdmin(const bool &adminValue)
```

Comment:
Optional fix. Be consistent in naming, setIsAdmin(const bool isAdmin)

Also don't use reference '&' for C++ primitive types

Reply:
+